<!DOCTYPE html>
<html lang="en">

<head>
    <title>MZKit | Data toolkits for processing NMR, MALDI MSI, MALDI single cell, Raman Spectroscopy, LC-MS and GC-MS
        raw data, chemoinformatics data analysis and data visualization.</title>
    <!-- Meta -->
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="shortcut icon" href="favicon.ico">
    <link rel="manifest" href="/manifest.json">
    <link href='https://fonts.googleapis.com/css?family=Lato:300,400,300italic,400italic' rel='stylesheet'
        type='text/css'>
    <link href='https://fonts.googleapis.com/css?family=Montserrat:400,700' rel='stylesheet' type='text/css'>
    <!-- FontAwesome JS-->
    <script defer src="assets/fontawesome/js/all.min.js"></script>
    <!-- Global CSS -->
    <link rel="stylesheet" href="assets/plugins/bootstrap/css/bootstrap.min.css">
    <!-- Plugins CSS -->
    <link rel="stylesheet" href="assets/plugins/prism/prism.css">
    <!-- Theme CSS -->
    <link id="theme-style" rel="stylesheet" href="assets/css/theme-1.css">
    <link id="theme-style" rel="stylesheet" href="assets/css/slider.css">
    <!-- GitHub Button -->
    <script async defer src="https://buttons.github.io/buttons.js"></script>

    <style>
        .card-text {
            font-size: 0.9em;
        }

        .docs .docs-inner {
            max-width: 1200px;
        }

        blockquote {

            border-left: #17baef 7px solid;
            background-color: #f3f3f3;
            color: #585858;
            padding: 0px 10px 1px 10px;

        }
    </style>

</head>

<body data-spy="scroll">

    <!-- ******HEADER****** -->
    <header id="header" class="header">
        <nav id="main-nav" class="main-nav navbar navbar-expand-md">
            <div class="container-fluid">
                <a class="navbar-brand logo" href="/">
                    <span class="logo-title">MZKit</span>
                </a>

                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar-collapse"
                    aria-controls="navbar-collapse" aria-expanded="false" aria-label="Toggle navigation">
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </button>
                <!--//nav-toggle-->

                <div class="navbar-collapse collapse" id="navbar-collapse">
                    <ul id="nav-menu" class="nav navbar-nav ms-md-auto">
                        <li class="nav-item sr-only"><a class="nav-link" href="/">Home</a></li>
                    </ul>
                    <!--//nav-->
                </div>
                <!--//navabr-collapse-->
            </div>
        </nav>
        <!--//main-nav-->
    </header>
    <!--//header-->

    <!-- ******PROMO****** -->
    <section id="promo" class="promo section offset-header">
        <br />
        <br />
        <br />

        <div class="container text-center banner">

            <h1 class="title" style="color: white;text-shadow: 2px 2px 32px #ffffff9c;">MZ<span
                    class="highlight">Kit</span>
            </h1>
            <p class="intro">Data toolkits for processing NMR, MALDI MSI, MALDI single cell, Raman Spectroscopy, LC-MS
                and GC-MS raw data, chemoinformatics data analysis and data visualization.</p>
        </div>

        <br />
        <br />
        <br />
        <br />
        <br />
        <br />
        <br />
    </section>
    <!--//promo-->



    <!-- ******DOCS****** -->
    <section id="docs" class="docs section">
        <div class="container">
            <div class="docs-inner">
                <h2 class="title text-center" id="tutorials">Update History</h2>

                <!--//block-->
                <div class="block">



                    <div class="row">

                        <div class="col">

                            <p>
                                Mzkit is an open source raw data file toolkit for mass spectrometry data analysis,
                                provides by the BioNovoGene corporation. The features of mzkit inlcudes: raw data file
                                content viewer(XIC/TIC/Mass spectrum plot), build molecule network, formula de-novo
                                search and de-novo annotation.
                            </p>
                            <p>
                                This open source mass spectrometry data toolkit is developed at the BioDeep R&D
                                laboratory and brought to you by BioNovoGene corporation.
                            </p>

                            <div>



















                                
        <h5 id="5.13.2024.0917">5.13.2024.0917</h5>
        <blockquote>
            
        <a href="https://doi.org/">
            <img src="https://zenodo.org/badge/DOI/.svg" alt="DOI of ver5.13.2024.0917">
        </a>
    
        </blockquote>
        <p>
            #Sep 17, 2024#
        </p>
        <ul>
            <li>
			<span class="badge rounded-pill bg-primary">new</span> 
			search biodeep database online resource for make a specific ion annotation in metabonomics workbench module
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			make improvements of the table viewer: add send to ms-imaging ion list function
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			make improvements of the spatial expression peaktable generator algorithm: make expression data bootstrapping in MSI data service backend(make bootstrapping in frontend UI in old release, this may crashed the entire workbench window if the data payload is too big)
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			make improvements of the metabonomics workbench module: add peaktable pre-processing, data selection, imports annotation, filter annotation function
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			make improvements of the ms-imaging region operation: enable extract region sample data from the loaded tissue region data.
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			enable make config of the adducts when do ms1 peak annotation search
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			make bug fixed for rgb overlaps ms-imaging in old release: when the number of selected ions is less than 3 ion, the export script will be crashed, now it has been fixed
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			make error fixed for load kegg compound database
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			make error fixed of the possible file stream not released when make ms-imaging peakfile exports, this may crashed the data export script background task process.
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			fixed of the un-expected raster data filter when do rgb ions overlaps ms-imaging plot exports
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			fix of the data reader of the kegg enrichment viewer: data table field name is mis-matched with the reader, this will carashed the entire workbench program in the old release.
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			fixed of config default XIC mass tolerance window in the workbench settings page
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			fix of the fragment annotation in lc-ms data viewer
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			enable run batch LC-MS rawdata file deconvolution for get a xcms format liked peaktable result, discussion on github: [Does this page is the result of peak picking? xcms_id?Not AccumulateROI? #30](https://github.com/xieguigang/mzkit/issues/30)
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			add new hatmap color palette to the ms-imaging render: ``Seismic`` and ``Icefire``
		 </li>
		
        </ul>

        
            <p>
                <blockquote>🎉🎇🎇2024-09-17<br />🎉Happy Mid-Autumn Festival!<br />🧨🧨🧨🎉🎉🎇🎇</blockquote>
            </p>
            
	
        <h5 id="5.9.3325.159">5.9.3325.159</h5>
        <blockquote>
            
        <a href="https://doi.org/">
            <img src="https://zenodo.org/badge/DOI/.svg" alt="DOI of ver5.9.3325.159">
        </a>
    
        </blockquote>
        <p>
            #Jun 29, 2024#
        </p>
        <ul>
            <li>
			<span class="badge rounded-pill bg-primary">new</span> 
			add single cells metabolomics data viewer module(in version 1)
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			fix of the tcp networking module in ms-imaging viewer([see details issue #28](https://github.com/xieguigang/mzkit/issues/28))
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			fix the login module for the new biodeep online database services
		 </li>
		
        </ul>

        
	
        <h5 id="5.2.7336.3219">5.2.7336.3219</h5>
        <blockquote>
            
        <a href="https://doi.org/">
            <img src="https://zenodo.org/badge/DOI/.svg" alt="DOI of ver5.2.7336.3219">
        </a>
    
        </blockquote>
        <p>
            #Feb 09, 2024#
        </p>
        <ul>
            <li>
			<span class="badge rounded-pill bg-primary">new</span> 
			simple workbench toolset for metabonomics data analysis and visualization
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			view GCxGC rawdata file
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			LC-MS reference spectrum library management
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			make improvements of the 3d ms-imaging model viewer: allowes load raw raster byte volume model
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			MoNA online services plugin for build local LCMS reference spectrum library
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			fix for load GCMS netCDF rawdata file(add new file reader based on the unidata netcdf-c library)
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			add supports of metabonomics data analysis for the selected interested ion layer in ms-imaging viewer
		 </li>
		
        </ul>

        
            <p>
                <blockquote>🎉🎇🎇2024<br />🎉Happy chinese new year!<br />🧨🧨🧨🎉🎉🎇🎇</blockquote>
            </p>
            
	
        <h5 id="4.8.199.3374">4.8.199.3374</h5>
        <blockquote>
            
        <a href="https://doi.org/10.5281/zenodo.8076924">
            <img src="https://zenodo.org/badge/DOI/10.5281/zenodo.8076924.svg" alt="DOI of ver4.8.199.3374">
        </a>
    
        </blockquote>
        <p>
            #Oct 15, 2023#
        </p>
        <ul>
            <li>
			<span class="badge rounded-pill bg-primary">new</span> 
			add a new color scaler palette pattern: Typhoon
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			Add median layer imaging for ms-imaging rawdata
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			enable tweaks of the more parameters when export ms-imaging plot file, example as: color map levels, raster filter pipeline, etc
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			Show the feature list of current mouse cursor selected spot in ms-imaging viewer
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			Show the mass spectrum property information of current mouse cursor selected spot in ms-imaging viewer
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			fix of the possible program crashed error when view a specific gene expression heatmap from the Spatial Transcriptome data
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			fix of the dzi index problem when view the MZKit hds pack in pathological virtual slide viewer(due to the reason of file name is mis-matched with the dzi index name, now this problem has been fixed)
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			add imports of the 10x Visium h5ad rawdata file, you can just set h5ad file source when imports processed 10x Visium rawdata file, or the additional gene symbol, barcode, spot index information is required when imports the space ranger output file.
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			make update the layout algorithm for display the physical ruler in ms-imaging viewer
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			make updates of the Mummichog algorithm, apply the Monte Carlo method for updates the metabolite annotation candidate list in Mummichog
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			make updates of the Moran index calculation for specific ion layer in ms-imaging rawdata processing
		 </li>
		
        </ul>

        
	
        <h5 id="4.7.31.9974">4.7.31.9974</h5>
        <blockquote>
            
        <a href="https://doi.org/10.5281/zenodo.8076924">
            <img src="https://zenodo.org/badge/DOI/10.5281/zenodo.8076924.svg" alt="DOI of ver4.7.31.9974">
        </a>
    
        </blockquote>
        <p>
            #Sep 29, 2023#
        </p>
        <ul>
            <li>
			<span class="badge rounded-pill bg-primary">new</span> 
			add a new ribbon menu button for open the pathological virtual slide file, you can access this new feature via menu [File] -> [Toolkit] -> [Open Slide]
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			add new function for do HE stain image overlap with the ms-imaging slide data
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			enable visualize the kegg enrichment result in network graph viewer, use this function just via access the [KEGG enrichment graph] function in the table visualization
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			add file format supports for open the AP-SMALDI udp project file in ms-imaging viewer
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			add LCMS ms1 scatter data viewer
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			add 3d viewer for the LCMS ms1 scatter data
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			add function for adjust the ms-imaging slide its position automatically, used this new function via the [Auto Location] menu button in the ms-imaging viewer
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			add more file format supports for the mzkit pathological virtual slide viewer, it is based on the openslide library for processing the slide file 
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			make updates of the pathological virtual slide viewer, you can make a screenshot, export deep zoom image package from the viewer
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			add splash-id for ms spectrum property display
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			add moran index and moran p-value for ms-imaging property viewer
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			You can tweaks of the ms-imaging render pipeline, via the [Tweaks MSI filters] menu button in the ms-imaging viewer.
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			make errors fixed for load KEGG compound library when run peak list annotation
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			Some of the menu icon of mzkit has been updates to Windows11 color icons, menu icon comes from the free icon library website: https://icons8.com/icons
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			Make updates of the ms-imaging plot export function, now you can adjust the plot image size, plot content padding and plot dpi for export both single ion ms-imaging plot and rgb overlaps ms-imaging plot
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			updates the demo pathological virtual slide deep zoom image file
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			add intensity quantile plot in the current layer mz histogram viewer
		 </li>
		
        </ul>

        
            <p>
                <blockquote>Chinese people, especially the poets, have a special emotion for the moon. The affections they hold may surpass those of any other nations in the world, forming a special moon sentiment. The sight of the moon can affect their emotions, and arouse their hopes and dreams for the future.<br />Happy Mid-autumn Festival!</blockquote>
            </p>
            
	
        <h5 id="4.5.144.3584-patch1">4.5.144.3584-patch1</h5>
        <blockquote>
            
        <a href="https://doi.org/10.5281/zenodo.8076924">
            <img src="https://zenodo.org/badge/DOI/10.5281/zenodo.8076924.svg" alt="DOI of ver4.5.144.3584-patch1">
        </a>
    
        </blockquote>
        <p>
            #Jun 26, 2023#
        </p>
        <ul>
            <li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			add context menu options for view molecular networking cluster spectrum in [molecular network viewer]
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			add black color into the FlexImaging color palette
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			make bugs fixed for load ms-imaging data from Thermo Fisher raw file
		 </li>
		
        </ul>

        
	
        <h5 id="4.5.139.9689">4.5.139.9689</h5>
        <blockquote>
            
        <a href="https://doi.org/10.5281/zenodo.8076924">
            <img src="https://zenodo.org/badge/DOI/10.5281/zenodo.8076924.svg" alt="DOI of ver4.5.139.9689">
        </a>
    
        </blockquote>
        <p>
            #Jun 24, 2023#
        </p>
        <ul>
            <li>
			<span class="badge rounded-pill bg-primary">new</span> 
			Enable mzkit for imports 10x genomics raw data file for do slide sample imaging
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			Enable mzkit for do ms-imaging slide sample rotation, just use the button [Rotate Slide] in MSI operations menu.
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			Add ion annotation table imports for ms-imaging viewer, this function required a csv table that contains the target ions' m/z, name, formula, precursor_type information.
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			Add a color scaler bar besides the ms-imaging viewer, you can adjust the intensity scaler range manually via this color scaler bar. This color scaler bar not working for the RGB ms-imaging render.
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			View ion m/z distribution histogram in ms-imaging viewer. You can found this option in [MSI Analysis] -> [Feature Detections] menu.
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			Add umap manifold + dbscan clustering for do slide sample auto-segmentation
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			Add app plugin for view mzkit LCMS reference spectrum library data pack file, use this function by open plugin from [Plugin Manager] -> [Reference Library Viewer]
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			Add app plugin for access to the Spectrum cluster tree database, use this function by open plugin from [Plugin Manager] -> [Molecular Network Viewer Client]
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			show physical ruler overlap in the ms-imaging render, you can turn off this overlap by set showPhysicalRuler option to false
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			show total ions ms-imaging overlaps in the ms-imaging render, you can turn off this overlap by set showTotalIonOverlap option to false
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			enable adjust the ion polarity mode for the ms-imaging data viewer, this option value will affects some ion metabolite annotation related function in mzkit ms-imaging data viewer
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			add more ms-imaging related properties to displayed in ms-imaging viewer: resolution, physical size in unit mm, file type, etc.
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			enable imports custom ion layer in the ion layer selector of the ms-imaging viewer, data imports could be comes from a csv or xlsx table file.
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			Add tweaks of the parameters when do ms-imaging row scan raw data file combines
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			enable set the color palette when create the ms-imaging heatmap matrix, you can just tweaks of this parameter in the [Settings] tab page on the ms-imaging heatmap matrix editor
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			Now the umap 3d scatter viewer could interactive with the ms-imaging viewer: you can select the ms-imaging pixel spot via mouse click on the scatter point in umap 3d scatter viewer
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			Enable the ms-imaging viewer imports the umap cluster dataset as the sample slide segmentation data source
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			Add scatter size options for tweaks the point size when view the tissue segmentation result overlap with the ms-imaging
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			make improvements of the targetted metabolism data viewer: export targetted linear modelling report PDF
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			Enable display the RGB ms-imaging renering plot by default when load plot data from the export cdf matrix file comes from the mzkit RGB ms-imging rendering result
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			fix of the possible bug in the background R# script data backend when create the ms-imaging heatmap matrix
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			removes the biodeep login condition for the targetted metabolism data handler
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			Add options for group the un-tagged pixel spots in the ms-imging slide sample in the ms-imaging tissue sample region editor
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			Add option for view sample region MS1 spectrum in the ms-imaging tissue sample region editor, use the mouse right click to invoke this function
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			Add option for set custom color in html color code for the ms-imaging tissue sample region
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			Add function for load all annotated molecule layer from the ms-imaging raw data file, you can use this new function via mouse contex menu of the ion layer selector
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			Add batch export of the ms-imaging layer image via mouse contex menu of the ion layer selector
		 </li>
		
        </ul>

        
            <p>
                <blockquote>This is a release update edition for PANOMIX spatial metabolism<br />Wish you peace and health on Chinese Dragon Boat Festival!</blockquote>
            </p>
            
	
        <h5 id="4.1.9.5689">4.1.9.5689</h5>
        <blockquote>
            
        <a href="https://doi.org/10.5281/zenodo.7553454">
            <img src="https://zenodo.org/badge/DOI/10.5281/zenodo.7553454.svg" alt="DOI of ver4.1.9.5689">
        </a>
    
        </blockquote>
        <p>
            #Jan 20, 2023#
        </p>
        <ul>
            <li>
			<span class="badge rounded-pill bg-primary">new</span> 
			Add mzkit background services process viewer
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			Reconstruct of the mzkit workbench desktop application framework for implements the plugin feature, the mzkit app sotre online services is comming soon!
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			Add custom action for view scatter 3d data on the table data.
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			View spatial metabolism tissue spot cluster data in 3d scatter plot view
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			Add new feature for connect to the cloud services for processing ms-imaging data analysis and visualization
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			Add a new feature for view virtual pathology slide in tiff, dzi or ndpi format!
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			View NMR raw data spectrum
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			View 3D MALDI scan data, MS-imaging in 3D
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			Add single cell metabolism data processing feature into the mzkit R# package!
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			Merge multiple ms-imaging sample data file into one larger slide, and you also can custom the sample location layout in a super easy way
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			Add an excited feature which you can make the spatial mapping between the spatial transcription sample slide and the spatial metabolism sample slide!
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			fixed for the possible null reference error and the index out of range error, make the MS-imaging viewer more stable
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			make improvements of the task progress UI
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			add copy text and save log output text function to the log text output window
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			Add data context menu in the spectrum matrix viewer: [Open in table view] for send the spectrum matrix to the table viewer for the further data analysis or visualization, and the [Save] menu item for export the selected spectrum matrix into an excel file.
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			Add shortcut button for launch the ms data file conversion tool
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			make improvements of the ms-imaging render output post image processing function
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			change the [install Mzkit] button from install package internally to calling the external pipeline script for initialize the R# runtime environment.
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			Add functional buttons/context menu to the peak finding page: save matrix, imports matrix and copy peak list data.
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			Add options for check sample row data normalization when do row sample data merge of the MS-imaging row scans.
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			Make bugs fixed for the web query cache system: empty cache could make the empty query result always returned, now it has been fixed!
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			make bugs fixed for the in-correct slide scan width and height when do background clean in manually.
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			make improvements of the MS-imaging viewer, update to v2.0 version: do zoom, rotation of the ms-imaging output
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			Add parameter adjustment for the peak finding algorithm page
		 </li>
		
        </ul>

        
            <p>
                <blockquote>🎉🎇🎇2023<br />🎉Happy chinese new year!<br />🧨🧨🧨🎉🎉🎇🎇</blockquote>
            </p>
            
	
        <h5 id="3.6.729.1547">3.6.729.1547</h5>
        <blockquote>
            
        <a href="https://doi.org/10.5281/zenodo.7040586">
            <img src="https://zenodo.org/badge/DOI/10.5281/zenodo.7040586.svg" alt="DOI of ver3.6.729.1547">
        </a>
    
        </blockquote>
        <p>
            #Nov 21, 2022#
        </p>
        <ul>
            <li>
			<span class="badge rounded-pill bg-primary">new</span> 
			SMILES molecule structure designer, power by <a href='https://lifescience.opensource.epam.com/ketcher/index.html'>Ketcher</a>!
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			CFM-ID workflow for run mass spectrum prediction in level2 based on the SMILES/InChI molecule structure string
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			add plot of the phenograph spot cluster information in ms-imaging viewer
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			improvements of the ms-imaging render parameter grid: add options for turn imaging filter and ruler display
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			make improvements of the workbench home page display
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			make improvements of the UI behaviour of the polygon editor for the ms-imaging sample
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			make bugs fixed for generates the gray scale color palette
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			fix of the possible app crashed problem when drawing an empty polygon in ms-imaging viewer
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			update to latest mzkit R# package
		 </li>
		
        </ul>

        
	
        <h5 id="3.5.593.3968">3.5.593.3968</h5>
        <blockquote>
            
        <a href="https://doi.org/10.5281/zenodo.7040586">
            <img src="https://zenodo.org/badge/DOI/10.5281/zenodo.7040586.svg" alt="DOI of ver3.5.593.3968">
        </a>
    
        </blockquote>
        <p>
            #Sep 01, 2022#
        </p>
        <ul>
            <li>
			<span class="badge rounded-pill bg-primary">new</span> 
			add tissue morphology map overlaps to MSI viewer
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			add tissue morphology map editor feature to create custom tissue region for data analysis
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			add HE stain image analysis module
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			add a new general table data viewer for open microsoft excel table files
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			add a general data visualization module for plot data based on the table viewer content
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			add new ggplot package for data plot pipeline task
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			add data visualization template rendering for MS-imaging plot
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			enable view multiple sample MS-imaging data
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			add online pubchem metabolite database query function for the ion feature in MSI raw data
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			add data compatibility with the bruker SCiLS lab software
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			make improvements of the ms1 peak list data annotation function
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			update internal metabolite database, extends database list from KEGG only to KEGG/lipidmaps/chebi
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			make the molecular networking viewer interactive
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			add ms1 peak deconvolution function to raw data viewer
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			add peak finding analysis feature to the general signal data analysis
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			new application installer experience
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			upgrade the internal Rstudio environment from .NET4.8 assembly to .NET6.0 assembly
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			add mzwork project file for share the workspace between the device
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			make improvements of the MRM/GCMS targetted data viewer
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			add fix patch script to install internal Rstudio environment
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			upgrade mzPack format to version 2.0, make improvements of the data compatibility between the single sample data and multiple sample data
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			make bugs fixed of the style tweaking for raw data plot viewer
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			handling of the data serialization error when raw data file its file path is a windows UNC path
		 </li>
		
        </ul>

        
	
        <h5 id="1.32.7743.6345">1.32.7743.6345</h5>
        <blockquote>
            
        <a href="https://doi.org/10.5281/zenodo.4603277">
            <img src="https://zenodo.org/badge/DOI/10.5281/zenodo.4603277.svg" alt="DOI of ver1.32.7743.6345">
        </a>
    
        </blockquote>
        <p>
            #Mar 14, 2021#
        </p>
        <ul>
            <li>
			<span class="badge rounded-pill bg-primary">new</span> 
			add peak annotation result for MS matrix viewer
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			targetted quantification linear modelling and sample quantify evaluation (required login BioDeep web services)
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			add 3D plot of LC-MSMS MRM ion TIC overlaps plot
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			add demo scripts into the file explorer
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			improvements of the raw data charting plot styles
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			improvements of the GCMS/LC-MSMS targetted file explorer
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			improvements of the GCMS feature ROI explorer
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			improvements of the document page model
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			add background task progress display on the status bar
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			fixed of R# script editor syntax highlight problem
		 </li>
		<li>
			<span class="badge rounded-pill bg-info text-dark">update</span> 
			update to latest mzkit R# package
		 </li>
		
        </ul>

        
	
        <h5 id="1.32.7692.29666">1.32.7692.29666</h5>
        <blockquote>
            
        <a href="https://doi.org/10.5281/zenodo.4456618">
            <img src="https://zenodo.org/badge/DOI/10.5281/zenodo.4456618.svg" alt="DOI of ver1.32.7692.29666">
        </a>
    
        </blockquote>
        <p>
            #Jan 22, 2021#
        </p>
        <ul>
            <li>
			<span class="badge rounded-pill bg-primary">new</span> 
			MS-imaging
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			View MRM/GCMS SIM data files
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			Add plot tweaks function
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			Improvements of the R# scripting editor
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			Update R# interpreter engine
		 </li>
		
        </ul>

        
	
        <h5 id="v1.0.0.2-beta">v1.0.0.2-beta</h5>
        <blockquote>
            
        <a href="https://doi.org/10.5281/zenodo.4274582">
            <img src="https://zenodo.org/badge/DOI/10.5281/zenodo.4274582.svg" alt="DOI of verv1.0.0.2-beta">
        </a>
    
        </blockquote>
        <p>
            #Nov 15, 2020#
        </p>
        <ul>
            <li>
			<span class="badge rounded-pill bg-primary">new</span> 
			Mass spectrum similarity search
		 </li>
		<li>
			<span class="badge rounded-pill bg-dark">enhancement</span> 
			Improvements of the content project file model
		 </li>
		<li>
			<span class="badge rounded-pill bg-success">fixed</span> 
			Bugs fixed of the mzkit_win32 UI
		 </li>
		
        </ul>

        
	
        <h5 id="v1.0.0.1-beta">v1.0.0.1-beta</h5>
        <blockquote>
            
        <a href="https://doi.org/10.5281/zenodo.4091067">
            <img src="https://zenodo.org/badge/DOI/10.5281/zenodo.4091067.svg" alt="DOI of verv1.0.0.1-beta">
        </a>
    
        </blockquote>
        <p>
            #Oct 15, 2020#
        </p>
        <ul>
            <li>
			<span class="badge rounded-pill bg-primary">new</span> 
			View mzML/mzXML raw data files
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			Formula de-novol search
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			Molecular networking
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			Feature search
		 </li>
		<li>
			<span class="badge rounded-pill bg-primary">new</span> 
			R# scripting for mzkit automation
		 </li>
		
        </ul>

        
	



































                            </div>
                        </div>

                    </div>

                    <div class="row">

                        <div class="col">
                            <p><a href="HISTORY-zh.html">Chinese</a> Version</p>
                        </div>
                    </div>

                </div>
                <!--//block-->

            </div>
            <!--//docs-inner-->
        </div>
        <!--//container-->
    </section>
    <!--//features-->



    <!-- Javascript -->
    <script src="assets/plugins/bootstrap/js/bootstrap.min.js"></script>
    <script src="assets/plugins/prism/prism.js"></script>
    <script src="assets/plugins/gumshoe/gumshoe.polyfills.min.js"></script>
    <script src="assets/js/main.js"></script>
    <div style="width:100%;text-align: center;display: none;">
        <script type="text/javascript"
            src="//rf.revolvermaps.com/0/0/8.js?i=5vdm1y3dswl&amp;m=7&amp;c=ff0000&amp;cr1=ffffff&amp;f=georgia&amp;l=0&amp;s=175&amp;hi=70"
            async="async"></script>
    </div>
    <div style="width:100%;text-align: center;">
        <div style="width:300px;padding-left: 45%;padding-top: 20px;padding-bottom: 20px;">
            <script type='text/javascript' id='clustrmaps' src='//cdn.clustrmaps.com/map_v2.js?cl=080808&w=300&t=tt&d=3_C51iZSI0IgVEWjFQVnYDECJqMn-sUnXAJKTgdQBDM&co=ffffff&ct=808080&cmo=3acc3a&cmn=ff5353'>
            </script>
        </div>        
    </div>
    <div style="width: 100% ; text-align: center;">
        Copyright © 2013-2022 苏州帕诺米克生物医药科技有限公司 · 版权所有 | 中國 蘇州
    </div>

</body>

</html>
